modules/control/submodules/preprocessor_submodule.h

```diff
@@ -155,14 +148,15 @@ class PreprocessorSubmodule : public apollo::cyber::TimerComponent {
   std::shared_ptr<cyber::Reader<apollo::localization::LocalizationEstimate>>
       localization_reader_;
   std::shared_ptr<cyber::Reader<planning::ADCTrajectory>> trajectory_reader_;
-  std::shared_ptr<cyber::Writer<apollo::control::ControlCommand>>
-      control_command_writer_;
+
+  std::shared_ptr<cyber::Writer<apollo::control::Preprocessor>>
+      preprocessor_writer_;
```

Comment:
You don't need the `apollo::control::` prefix as you are already inside the namespace. Then it fits in one line.